### PR TITLE
Initial Release: 1.0.0 version of the website

### PR DESCRIPTION
**Pull request by isume295**
## What does this PR do?

This PR is raised to implement the following features 👍:

### API
- [x] Use an external API so you can base the development of the webapp around it. 
 
### Interface
- [x] Build *rocket page*,  *mission page* and *profile page* interfaces using the provided wireframes.

#### rocket page
- [x] When the page loads, the web app retrieves data from an external API.
- [x] User can click on the Reserve Rocket button to reserve a rocket
- [x] User can click on the  Cancel button to cancel the reservation.
 

#### mission page
- [x] When the page loads, the web app retrieves data from an external API.
- [x] User can click on the Join mission button to reserve a mission
- [x] User can click on the Leave mission button to cancel the reservation.

 
## Technical set up 🙌
- [x] Set up the repository on GitHub and use Gitflow.
- [x] Set up a JavaScript testing library (Jest).
- [x] Added the JSX for the structure. 
- [x] Added the styled components for the styling.
- [x] Used the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯

### CODE REVIEW

File: src/redux/rockets/rocketSlice.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/* eslint-disable array-callback-return */
```
Comment (Shisui6) [inline src/redux/rockets/rocketSlice.jsx]: Kindly go through your project and remove all lines that disable linter checks using the eslint disable statement. The linter checks are setup to help us learn best coding practices, and solving linter errors yourself will help you become a better developer😎

File: src/redux/rockets/rocketSlice.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  },
+  extraReducers: (builder) => {
+    builder
+      .addCase(getRockets.pending, (state) => ({
+        ...state,
+        isLoading: true,
+      }))
+      // .addCase(getRockets.fulfilled, (state, action) => {
+      //   console.log(action.payload);
+      // })
```
Comment (Shisui6) [inline src/redux/rockets/rocketSlice.jsx]: Kindly remove all commented-out code pieces of code from your project to keep your codebase clean🙏

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @isume295 and @Nasirkhan294, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- All functionality implemented ✅
- All tests are passing ✅
- Good job connecting react with redux ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/rockets/rocketSlice.jsx (hunk 1 above)
Comment (Nasirkhan294) [reply src/redux/rockets/rocketSlice.jsx]: Hi @Shisui6, Thanks for the constructive review, we will remove them.

File: src/redux/rockets/rocketSlice.jsx (hunk 2 above)
Comment (Nasirkhan294) [reply src/redux/rockets/rocketSlice.jsx]: Sure, we'll do that as well.

**Review by OyePriscilla — APPROVED** [review]
Hi @isume295 and @Nasirkhan294 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Removed commented-out lines of code ✔️ 
No linter checks disabled ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**